apps/main/src/dex/store/createQuickSwapSlice.ts

```diff
@@ -410,11 +409,9 @@ const createQuickSwapSlice = (set: SetState<State>, get: GetState<State>): Quick
 
       const { chainId, signerAddress } = curve
 
-      const selectToList = orderBy(
-        Object.entries(tokensMapper).map(([_, v]) => v!),
-        ({ volume }) => (typeof volume !== 'undefined' ? +volume : 0),
-        ['desc'],
-      ).map(({ address }) => address)
+      const selectToList = Object.entries(tokensMapper)
+        .map(([_, v]) => v!)
```

Comment:
This looks incorrect - if `tokensMapper` has no `null`/`undefined` why is it typed so?

Reply:
Probably because `export type TokensMapper = { [tokenAddress: string]: Token | undefined }`

If I remove the exclamation mark I actually get a type error that the `address` in `.map(({ address }) => address)` might be `undefined` as the type is `Token | undefined`. It's a bit silly because then I don't even know if you can have a key whose value is undefined?

Reply:
I feel like we're working around a typescript quirk here, but nonetheless I've added a `.filter((token) => !!token)`